package.json

```diff
@@ -11,9 +11,13 @@
   },
   "devDependencies": {
     "chai": "*",
-    "mocha": "*"
+    "mocha": "*",
+    "commander": "^2.9.0"
```

Comment:
This should go under "dependencies" since it's a library that will be use in prod.